CGPs/0033.md

```diff
@@ -42,8 +46,8 @@ https://explorer.celo.org/address/0x07F007d389883622Ef8D4d347b3f78007f28d8b7/rea
 
 
 ## Risks
-- Voting participation could be too sensitive to target voting yield changes. The value of adjustmentFactor=1/1825 could be too large in that case. However, the opposite has been the case so far: Voting yield has decreased from 0.06 to 0.053 and voting CELO fraction has been increasing from ~0.5 at epoch 50 to ~0.6 at epoch 400
-- Voting participation might not be sensitive enough. The value of adjustmentFactor=1/1825 could be too small in that case to effectively increase voting participation when it’s dropping. In that case, however, a small adjustmentFactor would be better than none at all, as it is currently.
+- Voting participation could be too sensitive to target voting yield changes. The value of 'adjustmentFactor' could be too large in that case. However, the opposite has been the case so far: Voting yield has decreased from 0.06 to 0.053 and voting CELO fraction has been increasing from ~0.5 at epoch 50 to ~0.6 at epoch 400
```

Comment:
use backticks for quoting code functions
`adjustmentFactor`

Reply:
thanks, resolved